examples/chip-tool/templates/commands.zapt

```diff
@@ -15,10 +18,97 @@
 #include <lib/support/BytesToHex.h>
 #include <lib/support/Span.h>
 #include <lib/support/TypeTraits.h>
+#include <lib/support/value_parser/ValueParserIntegral.h>
 #include <zap-generated/CHIPClientCallbacks.h>
 #include <zap-generated/CHIPClusters.h>
 
-// Value logging functions.  The non-generated ones depend on the
+namespace chip {
+
+{{#zcl_clusters}}
+{{#unless (isStrEqual name "Fixed Label")}}
```

Comment:
This is pretty fragile.  In particular if we end up with more structs shared across clusters (and media might do that), this will break.  The LogValue functions explicitly handle the shared structs separately from the non-shared structs, and it would be better to do that here too.

Followup fine for this change, though, in the insterests of getting this merged.

Reply:
Will add in follow-up.